src/Tester.js

```diff
@@ -70,6 +74,10 @@ export default class Tester extends Component {
     this.setState({key: Math.random()});
   }
 
+  clearAsync() {
```

Comment:
Needs to be an `async` function, returning a promise, for `await` to work.

Also double check `clear()` returns a promise (I think it does), or we'll have to wrap it in a promise generating function.